### PR TITLE
export colors as a hex values

### PR DESCRIPTION
**Pull request by bartoszlorek**
and now we will be able to import like that:
```javascript
import hex from 'style-guide/src/components/colors/hex';

hex.mintPrimary  // #53cf92
hex.mustardPrimary  // #fec83c
```

### CODE REVIEW

File: src/components/colors/hex.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import colors from './colors';
+
+const hex = {};
+
+Object.keys(colors).forEach(groupName => {
```
Comment (vergilius) [inline src/components/colors/hex.js]: I would export is as a function

**Comment by vergilius** [comment]
👍

File: src/components/colors/hex.js (hunk 1 above)
Comment (bartoszlorek) [reply src/components/colors/hex.js]: iife?

Comment (vergilius) [reply src/components/colors/hex.js]: nah, more like colors module could export function `getHexColors()` etc

but it's random thought, it's okay this way too

Comment (vergilius) [reply src/components/colors/hex.js]: It's a bit misleading, as colors already have `hex` property

**Pull request merged**